components/Button/Button.tsx

```diff
@@ -20,14 +19,42 @@ export const Button: React.FC<ButtonProps> = ({
   children,
   ...props
 }) => {
+  const isSmall = size === 'small';
+  const isMedium = size === 'medium';
+  const isLarge = size === 'large';
+  const classNames = cx(
+    {
+      'tw-py-1': isSmall,
+      'tw-px-2': isSmall,
+    },
+    {
+      'tw-py-2': isMedium,
+      'tw-px-4': isMedium,
+    },
+    {
+      'tw-py-4': isLarge,
+      'tw-px-8': isLarge,
+    },
```

Comment:
I wonder if this would be cleaner..

```suggestion
    {
      'tw-py-1 tw-px-2': isSmall,
      'tw-py-2 tw-px-4': isMedium,
      'tw-py-4 tw-px-8': isLarge,
    },
```

Reply:
Wow that's much cleaner! And you said that in way too passive of a voice lol, you're 110% correct.